### DISCUSSION TITLE
[Dev Question] Can i get source of the `livelyAudioListener` api ?

### DISCUSSION THREAD

**Discussion by TheArmagan** [rocksdanister/lively#486] [General]
Yeah i need the api for the `livelyAudioListener` i was trying for about one week now i tried making my own api on c# but i couldnt managed to do it. I am gonna use that API for my controlling RGB strip leds. Thanks :)

**Comment by rocksdanister** [comment]
https://github.com/filoe/cscore/tree/master/Samples/WinformsVisualization
